build.sh

```diff
@@ -230,7 +230,7 @@ if [ "$PIPELINE" = 'build' ]; then
     fi
     date
 elif [ "$PIPELINE" = 'update' ]; then
-    GAME_IMAGE="$GAME_IMAGE_LATEST"
+    GAME_IMAGE="$GAME_IMAGE_LAYERED"
     date
     if [ ! "$NO_PULL" = 'true' ]; then
         time docker pull "$GAME_IMAGE"
```

Comment:
`GAME_IMAGE="$GAME_IMAGE_LAYERED"` will not work because the image is pulled

Reply:
change to `docker pull "$GAME_IMAGE_LATEST"`?

Reply:
yes